src/wifipaf/tests/TestWiFiPAFLayer.cpp

```diff
@@ -349,6 +352,21 @@ TEST_F(TestWiFiPAFLayer, CheckRunAsCommissionee)
     packet_c1->AddToEnd(std::move(packet_c2));
     EXPECT_EQ(packet_c1->HasChainedBuffer(), true);
     EXPECT_EQ(newEndPoint->Send(std::move(packet_c1)), CHIP_NO_ERROR);
+    EXPECT_EQ(HandleWriteConfirmed(sessionInfo, true), CHIP_NO_ERROR);
+
+    // Test, Send the packet while resource is unavaialbe -> available
+    mResourceAvailable   = false;
+    auto packet_resource = System::PacketBufferHandle::NewWithData(buf_chain, sizeof(buf_chain));
+    EXPECT_EQ(newEndPoint->Send(std::move(packet_resource)), CHIP_NO_ERROR);
+    // break because resource is unavailable
+    EXPECT_EQ(isSendQueueNull(), false);
+    // Resource is available now
+    mResourceAvailable = true;
+    // PAF packets shoudl be sent within a second
+    sleep(2);
```

Comment:
sleep in unit tests are generally not desirable. Can you use test timers?

https://github.com/project-chip/connectedhomeip/blob/master/src/system/SystemClock.h#L350
https://github.com/project-chip/connectedhomeip/blob/master/src/system/SystemClock.h#L344